Unify Formatting (#82)
C2Devel/terraform-provider-croccloud: website/docs/r/eip.md

layout: "aws"
page_title: "aws_eip"
description: |-

>   Manages an Elastic IP resource.

Reviewer: ```suggestion
  Manages an Elastic IP.
```

И в описании тоже

The following arguments are supported:

* `address` - (Optional) IP address from an EC2 BYOIP pool.
    _Constraints:_ This option is only available for VPC EIPs
* `associate_with_private_ip` - (Optional) User-specified primary or secondary private IP address to associate with the elastic IP address.
    * _Constraints:_ If no private IP address is specified, the elastic IP address is associated with the primary private IP address

> * `instance` - (Optional) ID of the EC2 instance.

Reviewer: The ID of

* `network_interface` - (Optional) ID of the network interface to associate with.
* `public_ipv4_pool` - (Optional) EC2 IPv4 address pool identifier.

>     * _Constraints:_  This option is only available for VPC EIPs.

Reviewer: Без точки

The following arguments are supported:

* `address` - (Optional) IP address from an EC2 BYOIP pool.
    _Constraints:_ This option is only available for VPC EIPs
* `associate_with_private_ip` - (Optional) User-specified primary or secondary private IP address to associate with the elastic IP address.
    * _Constraints:_ If no private IP address is specified, the elastic IP address is associated with the primary private IP address
* `instance` - (Optional) The ID of the EC2 instance.
* `network_interface` - (Optional) The ID of the network interface to associate with.
* `public_ipv4_pool` - (Optional) The ID of the EC2 IPv4 address pool.

>     * _Constraints:_  This option is only available for VPC EIPs

Reviewer: ```suggestion
    * _Constraints_: This option is only available for VPC EIPs
```
двоеточие + двойной пробел

* `tags` - (Optional) Map of tags to assign to the EIP. If a provider [`default_tags` configuration block][default-tags] is used, tags with matching keys will overwrite those defined at the provider level.
    * _Constraints:_ Tags can only be applied to EIPs in a VPC
* `vpc` - (Optional) Boolean if the EIP is in a VPC or not.

~> **Note** You can specify either the ID of `instance` or the ID of `network_interface`, but not both.

~> **Note** If both `public_ipv4_pool` and `address` are specified, `address` will be used in the case both options are defined as API only requires one or the other.

## Attribute Reference

### Supported attributes

~> **Note** The data source computes the `public_dns` and `private_dns` attributes according to the [AWS VPC DNS Guide][vpc-dns-guide] as they are not available with the EC2 API.

In addition to all arguments above, the following attributes are exported:

* `allocation_id` - ID that the cloud assigns to represent the allocation of the elastic IP address for use with instances in a VPC.

> * `association_id` - ID representing the association of the address with an instance in a VPC.

Reviewer: Для этих ID можно использовать формулировки из пользовательской документации:
```suggestion
* `allocation_id` - The ID representing the allocation of the IP address.
* `association_id` - The ID representing the association of the allocation of the IP-address with an instance or a private IP address.
```

Author: Поправил, но тут какие-то древние формулировки (в пользовательской доке). Замучимся синхронизировать...

* `allocation_id` - ID that the cloud assigns to represent the allocation of the elastic IP address for use with instances in a VPC.
* `association_id` - ID representing the association of the address with an instance in a VPC.
* `domain` - Indicates if this EIP is for use in VPC (`vpc`).

> * `id` - Contains the ID of the EIP allocation.

Reviewer: ```suggestion
* `id` - The ID of the EIP allocation.
```

contains не нужен

* `private_ip` - Contains the private IP address. Can be `""` if `associate_with_private_ip` is specified.
* `public_ip` - Contains the public IP address.
* `tags_all` - Map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block][default-tags].

### Unsupported attributes

~> **Note** These attributes may be present in the `terraform.tfstate` file, but they have preset values and cannot be specified in configuration files.

The following attributes are not currently supported:

`carrier_ip`, `customer_owned_ip`, `customer_owned_ipv4_pool`, `network_border_group`, `private_dns`, `public_dns`.

> ~> **Note** The data source computes the `public_dns` and `private_dns` attributes according to the [AWS VPC DNS Guide][vpc-dns-guide] as they are not available with the EC2 API.

Reviewer: Перемести, пожалуйста, в Supported attributes
